recipes-wpe/wpeframework/wpeframework_git.bb

```diff
@@ -19,9 +19,6 @@ RPROVIDES_${PN} += "thunder"
 
 WPEFRAMEWORK_SYSTEM_PREFIX ??= "WPE"
 
-# Configure exceptions handling (ON/OFF)
-WPEFRAMEWORK_EXCEPTIONS_ENABLE ??= "OFF"
```

Comment:
What's the reason for deleting this flag?

Reply:
It is already available as part of PACKAGECONFIG, hence removed it